recipes-kernel/linux/linux-mainline/0010-sun8i-h2-plus-orangepi-zero.dts-disable-mmc1.patch

```diff
@@ -0,0 +1,26 @@
+From 2060e08623dd97b47f5c03651ac865d2efbd203a Mon Sep 17 00:00:00 2001
+From: Pawel Langowski <pawel.langowski@3mdeb.com>
+Date: Tue, 29 Oct 2024 09:47:51 +0100
+Subject: [PATCH] Disable wifi chip
+
+Signed-off-by: Pawel Langowski <pawel.langowski@3mdeb.com>
```

Comment:
Upstream-Status could be added

Reply:
We probably need to add status to all patches. I wonder why build didn't error out as it usually does without Upstream-Status

Reply:
[b089531](https://github.com/3mdeb/meta-rte/pull/73/commits/b089531c326b298eb1392b134a70cf4768ad12dc)

Reply:
It's not really oe-specific, rather RTE specific, but ok